README.md

```diff
@@ -40,3 +40,12 @@ Please refer to the instructions mentioned in the following [doc](./docs/addnewp
 ## Contribution
 
 This project uses [the Apache 2.0 license](./LICENSE). Contribution to this project requires the [DCO 1.1](./DCO1.1.txt) process to be followed.
+
+## Collaborations
+
+Slack: [#confidential-containers-peerpod](https://cloud-native.slack.com/archives/C04A2EJ70BX)  
+Zoom meeting: https://zoom.us/j/97726916398 on each `Wednesday` at.
+- 9:00  -- 10:00 GMT Winter time
```

Comment:
```suggestion
-  8:00 - 9:00 UTC
```

Reply:
Thanks @knrt10 and @stevenhorsman , fixed.